tests/integration_tests/test_examples.py

```diff
@@ -1,39 +1,84 @@
 """
```

Comment:
i think you could make this file a lot simpler if you defined glob strings for the different types of examples and wrote a test case for each glob string.

conditionals inside test cases is a bad code sell. if a file needs to be handled differently, then it's a different test case.